### DISCUSSION TITLE
Is it possible to set Web expiration time

### DISCUSSION THREAD

**Discussion by saul-hu** [drakkan/sftpgo#396] [Q&A]
Hi @drakkan. I have a question is it is possible to set Web expiration time?
Thanks in advance!

**Comment by drakkan** [comment]
Hi,

do you mean the cookie expiration time? The cookie duration is 10 minutes and it is refreshed each time you load a web page, so you will be logged out if you are inactive for about10 minutes

**Reply by saul-hu** [reply]
Thank you for your reply。
yes，Is it configurable ？

**Reply by drakkan** [reply]
Hi, no this is currently not configurable, what value would be appropriate for your use case? Even I make it configurable I don't want to allow very long token durations
